.github/workflows/backport.yml

```diff
@@ -29,12 +29,18 @@ jobs:
           persist-credentials: false
       - name: Install Actions
         run: npm install --production --prefix ./actions
+      - name: Get Github App secrets from vault
+        uses: grafana/shared-workflows/actions/get-vault-secrets@get-vault-secrets-v1.1.0
+        with:
+          repo_secrets: |
```

Comment:
let's explore using common_secrets, see more: https://github.com/grafana/shared-workflows/tree/main/actions/get-vault-secrets#examples

Reply:
we are using `repo_secrets`